src/styles.scss

```diff
@@ -179,4 +179,21 @@ body {
     overflow: visible;
   }
 
+  .genderRatingProgressBar .mat-progress-bar-buffer {
+    background-color: $error-red;
+  } 
+
+  .genderIconFemale {
+    float: right;
+    color: $error-red;
+  }
+
+  .genderIconMale {
+    float: left;
+    color: #2196f3;
+  }
+
+  .genderIconFemale {

```

Comment:
This is a repeated style (already set on line 188)

Reply:
Deleted repeated Css rule

Thanks @paulbert